src/app/invest/invest.component.html

```diff
@@ -1,71 +1,83 @@
 <div *ngIf="{
     wallet: wallet$ | async,
     project: project$ | async
-} as states" class="container root-content-container">
+} as states" class="container">
```

Comment:
even though this is ngIf, this won't wait for wallet and project to resolve. You need to assure with another `ngIf` that they exist. I'll comment it down there.